doc/develop/getting_started/index.rst

```diff
@@ -76,8 +76,11 @@ The current minimum required version for the main dependencies are:
 
       .. _install_dependencies_ubuntu:
 
-      #. Download, inspect and execute the Kitware archive script to add the
-         Kitware APT repository to your sources list.
+      #. If using an Ubuntu verion older than 22.04, it is necessary to add extra
```

Comment:
```suggestion
      #. If using an Ubuntu version older than 22.04, it is necessary to add extra
```

Reply:
I read it a few times and didn't spot the typo. Thanks!